packages/core/src/utils/editor.ts

```diff
@@ -134,9 +133,28 @@ export function getDiffCommand(
           'wincmd h | setlocal statusline=OLD\\ FILE',
           '-c',
           'wincmd l | setlocal statusline=%#StatusBold#NEW\\ FILE\\ :wqa(save\\ &\\ quit)\\ \\|\\ i/esc(toggle\\ edit\\ mode)',
-          // Auto close all windows when one is closed
+          oldPath,
+          newPath,
+        ],
+      };
+    case 'neovim':
+      return {
```

Comment:
Nit: this looks good, i think since we're modifying the behavior from what it was ( now users that use `:wq` will have to close out readonly file or use `:wqa`) i think it would be a good idea to call this out in the documentation, just calling out that if you're using the neovim or vim editor to use `:wqa`